opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -235,28 +235,28 @@ class SimulatorFullyImplicitBlackoilEbos
             // give the polymer and surfactant simulators the chance to do their stuff
             handleAdditionalWellInflow(timer, wells_manager, well_state, wells);
 
+            // Compute reservoir volumes for RESV controls.
+            computeRESV(timer.currentStepNum(), wells, state, well_state);
+
+            // Run a multiple steps of the solver depending on the time step control.
+            solver_timer.start();
+
+            const WellModel well_model(wells, &(wells_manager.wellCollection()), model_param_, terminal_output_);
+
+            auto solver = createSolver(well_model);
+
             // write the inital state at the report stage
             if (timer.initialStep()) {
                 Dune::Timer perfTimer;
                 perfTimer.start();
 
                 // No per cell data is written for initial step, but will be
                 // for subsequent steps, when we have started simulating
-                output_writer_.writeTimeStepWithoutCellProperties( timer, state, well_state );
```

Comment:
I guess the `output_writer_.writeTimeStepWithoutCellProperties()` method is now unused? If yes, mind removing it from the output writer class? (that class is flow_ebos specific, so flow_legacy is not affected by this change.)